src/OpenLoco/src/Audio/Audio.cpp

```diff
@@ -1043,7 +1011,7 @@ namespace OpenLoco::Audio
             case MusicPlaylistType::currentEra:
             {
                 auto currentYear = getCurrentYear();
-                auto info = kMusicInfo[_currentSong];
+                auto info = Jukebox::getMusicInfo(_currentSong);
```

Comment:
Oops, I think this was making a copy. Could you change this to:
```suggestion
                auto& info = Jukebox::getMusicInfo(_currentSong);
```